### PR TITLE
[ADD] migration for sale_service

### PR DESCRIPTION
**Pull request by hbrunn**
(formerly known as project_mrp)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: Please also add the openupgrade_analysis_work file

File: addons/sale_service/migrations/8.0.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.openupgrade import openupgrade
```
Comment (StefanRijnhart) [inline addons/sale_service/migrations/8.0.1.0/post-migration.py]: In new scripts, please import from openupgradelib

**Comment by hbrunn** [comment]
@StefanRijnhart @pedrobaeza done

**Comment by pedrobaeza** [comment]
Again :+1:

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Pull request merged**